src/adapter/src/coord/sequencer/inner.rs

```diff
@@ -1252,6 +1252,10 @@ impl Coordinator {
         }
         for dropped_in_use_index in dropped_in_use_indexes {
             session.add_notice(AdapterNotice::DroppedInUseIndex(dropped_in_use_index));
+            self.metrics
+                .optimization_notices
+                .with_label_values(&["DroppedInUseIndex"])
```

Comment:
I'm not sure if adding a value should be preferred over adding a separate metric per notice type. cc @teskje what would be better for minimizing the number of metrics?

Reply:
the # of time series will be the same either way (the name && labels together are uniquely identifying). what's here looks great